Makefile

```diff
@@ -102,9 +102,13 @@ else ifeq ($(CHAIN),callisto)
 APP_LOAD_PARAMS += --path "44'/820'"
 DEFINES += CHAINID_UPCASE=\"CALLISTO\" CHAINID_COINNAME=\"CLO\" CHAIN_KIND=CHAIN_KIND_CALLISTO CHAIN_ID=820
 APPNAME = "Callisto"
+else ifeq ($(CHAIN),aquachain)
+APP_LOAD_PARAMS += --path "44'/60'"
```

Comment:
wrong `"44'/60'"` entry.
no AQUA entry found at https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
@hackmod  thank you for checking this out. have been using `44'/60` (same as ETH) for all wallets, but i guess changing it sooner would be better than later. And, this would not break existing wallets.

also, this PR mainly got delayed only by lack of graphics, If you could point me towards "icon how-to", that would be greatly appreciated! Maybe there is some sort of script you use to make them all look the same.

will work on slip PR, rebase, and new HD path

Reply:
Rebased, New HD path set to chain ID (`44'/61717561'`)